py/symbolic/debuginfo.py

```diff
@@ -96,6 +96,16 @@ def debug_kind(self):
         """The kind of debug information in this object."""
         return str(decode_str(self._methodcall(lib.symbolic_object_get_debug_kind)))
 
+    @property
+    def features(self):
+        """The list of features offered by this debug file."""
+        features = set()
+        struct = self._methodcall(lib.symbolic_object_get_features)
+        for i in range(0, struct.len):
```

Comment:
Doesn’t matter too much but if you use a generator expression here you can construct a frozenset right away instead of making another throwaway object.